internal/locald/rootmanager/tp_monitor.go

```diff
@@ -128,16 +129,45 @@ func (mon *tpMonitor) checkTunnelProxyAccess(ctx context.Context) bool {
 			restartSvcs = true
 		}
 	}
+	if !restartSvcs {
+		rootMgr := mon.root.root
+		if rootMgr == nil {
+			return false
+		}
+		if rootMgr.localnetSVC == nil || rootMgr.localnetSVC.Status().Healthy {
```

Comment:
It should be `!rootMgr.localnetSVC.Status().Healthy`, isn't it?

Reply:
oops, thx!